sensing/autoware_pointcloud_preprocessor/src/downsample_filter/random_downsample_filter_node.cpp

```diff
@@ -58,7 +58,7 @@ RandomDownsampleFilterComponent::RandomDownsampleFilterComponent(
 {
   // set initial parameters
   {
-    sample_num_ = static_cast<size_t>(declare_parameter("sample_num", 1500));
```

Comment:
types do no coincide

Reply:
@knzo25 thanks! However, declare_parameter<size_t> doesn't work for ros2 parameter. (compile will fatiled)
![image](https://github.com/user-attachments/assets/b53d23ea-454b-4bc3-858a-4f7d1c28ec04)

reference: https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/

Reply:
Same reply as other PRs

Reply:
thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8485/commits/9d3fb7ff15c1fbd004c546b3f6db3e361b99a2dc